### DISCUSSION TITLE
Regarding airfoil motion

### DISCUSSION THREAD

**Discussion by tolstoydebbarma** [xcompact3d/Incompact3d#254] [Q&A] [answered]
Hi all, I want to simulate an airfoil NACA 0012 in Incompact3d. I want to give flapping motion to the trailing edge of the airfoil. 
![geom](https://github.com/xcompact3d/Incompact3d/assets/49072197/d72da348-9026-4ddd-bc20-be09e66f36ad)

Do I need to change this subroutine? If I give ubcx = -0.5, ubcy = 0 and ubcz =0, what will be the next step?

**Comment by slaizet** [comment, marked as answer]
Hi, for a static flapping airfoil (static = does not move in space except the flapping). ubcx=ubcy=ubcz=0 as the airfoil is static. My advice would be to start with the cylinder test case, replace the cylinder by a NACA profile and then make it flap.
Best,
Sylvain
